server/include/external/libxml/DOCBparser.h

```diff
@@ -0,0 +1,96 @@
+/*
+ * Summary: old DocBook SGML parser
+ * Description: interface for a DocBook SGML non-verifying parser
+ * This code is DEPRECATED, and should not be used anymore.
```

Comment:
Are we using this code? If so, this file says we shouldn't! :laughing:

Reply:
Yeh i am sure about this. There are several header files that are not used at the moment but are part of the libxm2 library package. My fear is that removing such files may contain necessary variables that will be needed by the library in the future.
There could be another way with the CMakeLists file. I can look into that.